admin-manual/maintenance/maintenance.rst

```diff
@@ -343,39 +342,188 @@ edit a configuration file from the command line.
 
 5. Restart Apache
 
-.. code:: bash
+   .. code:: bash
 
-    sudo apache2ctl restart
+      sudo apache2ctl restart
 
 6. Reload the dashboard page reporting the error in your browse
 
 7. Debug or report error
 
 8. Restore DEBUG to False and restart Apache to turn error reporting off again
 
-.. _browser-compatibility:
-
-Browser compatability
-^^^^^^^^^^^^^^^^^^^^^
+Resolve hanging decisions
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Archivematica has been tested most extensively with Firefox and Chrome. There are
-known issues with Internet Explorer 11 which result in an inability to start
-transfers in the dashboard (`Issue 7246 <https://projects.artefactual.com/issues/7246>`_). Minimal, but successful,
-testing has been done with Microsoft Edge.
+"My Transfer or Ingest tab has a red circle indicating that a decision is awaiting
+user input, but there are no decisions to be made on the screen. How do I get
+rid of the red circle?"
 
+.. image:: images/user-input.*
+   :align: center
+   :width: 80%
+   :alt: A red bubble on the Transfer tab at the top of the Archivematica dashboard indicates that a decision is awaiting user input.
+
+Any time a decision in the Transfer or Ingest tab is awaiting user input,
+Archivematica visually indicates the need for user input by placing a red circle
+on the tab. The numeral inside the circle indicates how many decisions are
+awaiting user input on the tab.
+
+It is possible to `remove transfers or SIPs`_ from the Transfer or Ingest tab
+without resolving a decision, which results in the red circle remaining even if
+there is no decision awaiting user input on the tab. While this isn't a critical
+issue, it is annoying. To get rid of the red circle, you need to use a command
+line tool to resolve the user input first and then either complete or reject the
+transfer/SIP. In the Archivematica source code, there is a python module
+containing an RPC client that can give you information about the transfers and
+SIPs that you have in your dashboard. It also allows you to add user inputs, as
+an alternative to using the dashboard.
+
+.. note::
+
+  If you are running Archivematica on CentOS/Redhat, you must first clone the
+  `archivematica-devtools`_ repo to use this and other tools.
+
+1. Navigate to the Archivematica MCP Client directory.
+
+   Ubuntu:
+
+      .. code-block:: console
+
+         sudo su - archivematica -s /bin/bash
+         source  /usr/share/archivematica/virtualenvs/archivematica-mcp-client/activate
+         export $(grep -v ^# /etc/default/archivematica-mcp-client)
+         cd /opt/archivematica/archivematica-devtools/bin/
+         ./am mcp-rpc-cli
+
+   CentOS:
+
+      .. code-block:: console
+
+         sudo su - archivematica -s /bin/bash
+         source  /usr/share/archivematica/virtualenvs/archivematica-mcp-client/activate
+         export $(grep -v ^# /etc/sysconfig/archivematica-mcp-client)
+         cd /opt/archivematica/archivematica-devtools/bin/
+         ./am mcp-rpc-cli
```

Comment:
@sallain @mamedin is this snippet going to work for both Ansible and package-based deploys?